src/Resources/contao/classes/DataContainer.php

```diff
@@ -273,6 +273,12 @@ protected function row($strPalette=null)
 			$this->varValue = \StringUtil::insertTagToSrc($this->varValue);
 		}
 
+		// Use raw request if set globally
+		if (isset($GLOBALS['TL_DCA'][$this->strTable]['config']['useRawRequestData']) && $GLOBALS['TL_DCA'][$this->strTable]['config']['useRawRequestData'] === true)
```

Comment:
Should we add a `!isset($arrData['eval']['useRawRequestData'])` check here too? This would make it possible to use the raw request for the whole DCA but opt-out of it for single fields.